core/src/main/scala/org/apache/spark/ui/scope/RDDOperationGraph.scala

```diff
@@ -246,8 +254,14 @@ private[spark] object RDDOperationGraph extends Logging {
     } else {
       ""
     }
+    val outputDeterministicLevel = node.outputDeterministicLevel match {
+      case DeterministicLevel.DETERMINATE => ""
+      case DeterministicLevel.INDETERMINATE => " [indeterminate]"
+      case DeterministicLevel.UNORDERED => " [unordered]"
+    }
```

Comment:
To be consistent with `[Barrier]` and `[Cached]`, t would be better to capitalize them.

Reply:
Thanks for the suggestion. Changed to capitalized ones.